events/utils.py

```diff
@@ -285,3 +286,26 @@ def get_user_data_source_and_organization_from_request(
         if publisher and publisher.replaced_by:
             publisher = publisher.replaced_by
     return data_source, publisher
+
+
+def clean_text_fields(
+    data,
+    allowed_html_fields: Optional[Iterable[str]] = None,
+    strip: Optional[bool] = False,
```

Comment:
```suggestion
    strip: bool = False,
```
`Optional` is a type hint for cases like `str | None` (https://docs.python.org/3/library/typing.html#typing.Optional)

Reply:
Fixed in the latest commit